src/ess/livedata/dashboard/scipp_to_holoviews.py

```diff
@@ -66,6 +66,28 @@ def convert_curve_1d(data: sc.DataArray) -> hv.Curve:
     return hv.Curve(data=(coord.values, data.values), kdims=kdims, vdims=vdims)
 
 
+def convert_error_bars_1d(data: sc.DataArray) -> hv.ErrorBars:
+    """
+    Convert a 1D scipp DataArray to a Holoviews Curve.
+
+    Returns
+    -------
+    hv.ErrorBars
+        A Holoviews ErrorBars object.
+    """
+    data = _ensure_coords(data)
+    dim = data.dim
+    coord = data.coords[dim]
+    kdims = [coord_to_dimension(coord)]
+    vdims = [create_value_dimension(data)]
+
+    return hv.ErrorBars(
```

Comment:
Is it possible to add markers for the means? The plot you posted only shows the error bars, not the mean values.

Reply:
As far as I can tell this is done by composing with, e.g., `hv.Curve`, but the styling (color) is different and I did not want to bother with that right now.